source/extensions/filters/http/oauth2/filter.cc

```diff
@@ -488,6 +512,15 @@ void OAuth2Filter::onGetAccessTokenSuccess(const std::string& access_code,
   finishGetAccessTokenFlow();
 }
 
+void OAuth2Filter::onUpdateAccessTokenSuccess(const std::string& access_code,
```

Comment:
Probably `onRefreshAccessTokenSuccess` is a better name (?).

Reply:
I agree, your name is better